### PR TITLE
NXBT-2815: Use Java 11 in docker images

### PR DESCRIPTION
**Pull request by kevinleturc**

### CODE REVIEW

**Comment by dmetzler** [comment]
This file is normally generated by using the `update.py` script.
You have to update the script so that it only updates the `master` and `>10.10` version.

File: templates/Dockerfile.centos [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
-# install java
-RUN yum install -y \
-       java-1.8.0-openjdk java-1.8.0-openjdk-devel 
+# install wget / tar
+RUN yum -y install wget tar
 
-# install wget
-RUN yum -y install wget 
+# install java
+RUN wget https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz
```
Comment (dmetzler) [inline templates/Dockerfile.centos]: - You must merge all the RUN commands
- you must verify the MD5 of the `tar.gz` (see how we do it when downloading the Nuxeo zip file)

File: templates/Dockerfile.rhel [hunk 2]
```diff
@@ -20,7 +20,7 @@ LABEL name="nuxeo/nuxeo" \
 # install java
 RUN yum-config-manager --disable rhel-7-server-htb-rpms && \
     yum install -y \
-       java-1.8.0-openjdk java-1.8.0-openjdk-devel wget unzip
+       java-11-openjdk wget unzip
```
Comment (dmetzler) [inline templates/Dockerfile.rhel:23]: Don't we need a full JDK and not a simple JRE? or did the RHEL packaging changed?

File: templates/Dockerfile.centos [hunk 3]
```diff
@@ -29,4 +31,4 @@ RUN yum -y upgrade && yum -y install \
     perl-Image-ExifTool \
     ghostscript
 
-%%RUN%%
\ No newline at end of file
+%%RUN%%
```
Comment (dmetzler) [inline templates/Dockerfile.centos]: Do cleanups in a separate commit

File: templates/Dockerfile.rhel [hunk 4]
```diff
@@ -54,4 +54,4 @@ RUN yum -y upgrade && yum -y install \
 ADD licenses /licenses
 ADD help.1 /help.1
 
-%%RUN%%
\ No newline at end of file
+%%RUN%%
```
Comment (dmetzler) [inline templates/Dockerfile.rhel]: Do cleanups in a separate commit

File: templates/pre-11.1/Dockerfile.centos [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    ffmpeg \
+    ffmpeg2theora \
+    ufraw \
+    poppler-utils \
+    libreoffice \
+    libwpd-tools \
+    perl-Image-ExifTool \
+    ghostscript
+
+%%RUN%%
```
Comment (dmetzler) [inline templates/pre-11.1/Dockerfile.centos:32]: Add an empty line :-) at the EOF

File: templates/pre-11.1/Dockerfile.rhel [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    libreoffice-core  \
+    libwpd-tools \
+    perl-Image-ExifTool \
+    ghostscript \
+    && yum clean all
+
+ADD licenses /licenses
+ADD help.1 /help.1
+
+%%RUN%%
```
Comment (dmetzler) [inline templates/pre-11.1/Dockerfile.rhel:57]: Add an empty line :-) at the EOF

**Review by dmetzler — CHANGES_REQUESTED** [review]
For the empty lines at EOF, if we include them in new templates, we can question if we want to propagate that change in previous `Dockerfile`. IMHO it would be more coherent but we have to do it in a separate commit as it may impact a lot of files after the update.

File: templates/Dockerfile.rhel (hunk 2 above)
Comment (kevinleturc) [reply templates/Dockerfile.rhel:23]: There's no more JRE.

**Comment by kevinleturc** [comment]
As it is impacting, I won't add new line at end of file.

**Review by dmetzler — APPROVED** [review]
